apps/main/src/llamalend/features/market-list/filters/RangeSliderFilter.tsx

```diff
@@ -42,30 +69,75 @@ export const RangeSliderFilter = <T,>({
   id: LlamaMarketColumnId
   format: (value: number) => string
   defaultMinimum?: number
+  adornmentVariant?: AdornmentVariants
+  scale?: 'power'
 }) => {
+  const isMobile = useIsMobile()
+  const minValue = 0
   const maxValue = useMemo(() => Math.ceil(getMaxValueFromData(data, field)), [data, field]) // todo: round this to a nice number
   const step = useMemo(() => Math.ceil(+maxValue.toPrecision(2) / 100), [maxValue])
-  const defaultValue = useMemo((): NumberRange => {
+  const isPowerScale = scale === 'power'
+  /** Find a power exponent that feels "good" based on the max value  */
+  const powerExponent = useMemo(() => {
+    if (!isPowerScale || maxValue <= minValue) return undefined
+    const safeMax = Math.max(maxValue, 1)
+    const exponent = Math.trunc(Math.log10(safeMax) - 2)
+    return exponent > 1 ? exponent : 1
+  }, [isPowerScale, maxValue, minValue])
+
+  const canUsePowerScale = isPowerScale && powerExponent != null
+
+  /** Helpers that convert real values to and from the slider's value space. */
+  const sliderValueTransform = useMemo(() => {
```

Comment:
My gut feeling tells me this logic to calculate `sliderValueTransform` can be made pur and moved outside of the component to the global level of this file. That function's only two parameters would be `minValue`, `maxValue` and `isPowerScale`. Then we'd have something like `const sliderValueTransform = useMemo(() => calculateSliderValueTransform(minValue, maxValue, scale === 'power'), [minValue, maxValue, scale])`